### PR TITLE
Fix - blanks and margin, buttons modal media requests

### PR DESCRIPTION
**Pull request by alex-bu-93**

### CODE REVIEW

**Comment by AleksandrBu** [comment]
image
image
image
image
image
image

File: apps/codelab/src/app/codelabs/angular/templates/templates.component.html [hunk 1]
```diff
@@ -330,7 +330,7 @@ <h2 i18n>
       [exercise]="code.ngForDirective.template"
       [highlights]="code.ngForDirective.matches.ngFor"
     ></codelab-exercise-playground>
-    <div class="info" i18n>
+    <div class="info" style="margin-top: 60px;" i18n>
```
Comment (kirjs) [inline apps/codelab/src/app/codelabs/angular/templates/templates.component.html]: Move to CSS?

Comment (AleksandrBu) [reply apps/codelab/src/app/codelabs/angular/templates/templates.component.html]: Done!)

File: libs/utils/src/lib/button-with-menu/button-with-menu.component.ts [hunk 2]
```diff
@@ -27,20 +27,26 @@ export class ButtonWithMenuComponent implements OnDestroy {
     private _vcr: ViewContainerRef,
```
Comment (aziz512) [inline libs/utils/src/lib/button-with-menu/button-with-menu.component.ts]: I think this prop needs a more explicit name

Comment (alex-bu-93) [reply libs/utils/src/lib/button-with-menu/button-with-menu.component.ts]: Usually '_vcr' is used, but sometimes I met 'container' name. How do you think, is it gonna be better?)

Comment (NothingEverHappens) [reply libs/utils/src/lib/button-with-menu/button-with-menu.component.ts]: vcr sounds good enough, prob no `_` prefix. But I'd search across the app and make it consistent.

**Pull request merged**